src/main/java/com/qunite/api/web/controller/QueueController.java

```diff
@@ -76,6 +79,34 @@ public ResponseEntity<Integer> memberPosition(@PathVariable(value = "id") Long q
     return ResponseEntity.of(queueService.getMemberPositionInQueue(memberId, queueId));
   }
 
+  @PostMapping("/{id}/members/{member-id}")
+  @Operation(summary = "Enroll member to queue")
+  public ResponseEntity<Void> enrollMemberToQueue(@PathVariable(value = "id") Long queueId,
+                                                  @PathVariable(value = "member-id")
```

Comment:
@PathVariable could be simplified as in membersAmount(@PathVariable Long id) method above